.github/workflows/build.yml

```diff
@@ -241,6 +242,112 @@ jobs:
           name: ${{ env.DEB_PACKAGE }}
           path: out/${{ env.DEB_PACKAGE }}
 
+  build-windows:
```

Comment:
The build job itself builds go binaries and upload zip files for various architectures in Windows. Do we need this separate job doing the same on a windows runner?

Reply:
yes. this job has run-on `windows-2019` while others is running on linux machines.

Reply:
Why is there a strict requirement to run this on windows? Can't we use the same zip file that got generated from the previous job?

Reply:
removed build-windows job https://github.com/hashicorp/consul-k8s/pull/2703/commits/5c3abb022c5c9692cd5a0af7098a39f57050bde8